server/src/db/migrations/20210201215646-create-rig.js

```diff
@@ -5,9 +5,11 @@ module.exports = {
       id: {
         primaryKey: true,
         type: Sequelize.STRING,
+        allowNull: false,
```

Comment:
I would advise caution when modifying existing migrations. Any machine where this migration has already ran will not run the modified migration.

Reply:
There's a database setup script (migrations.js) that undoes all migrations and seeders and then reruns all of them and the seeders each time the server restarts. Do agree that isn't the long term plan, but it has helped with testing.